client/core/core_test.go

```diff
@@ -7558,7 +7539,8 @@ func TestPreimageSync(t *testing.T) {
 		lo := convertMsgLimitOrder(msgOrder)
 		resp := orderResponse(msg.ID, msgOrder, lo, false, false, false)
 		limitRouteProcessing <- lo.ID()
-		<-time.NewTimer(time.Millisecond * 100).C
+		commitSig <- lo.Commit
+		time.Sleep(200 * time.Millisecond)
```

Comment:
I commented out this line and ran 
`parallel -P 4 -N0 --halt soon,fail=1 go test -race -count=1 -short -run TestPreimageSync :::: <(seq 1 50)`
and had no fails.